clients/hadoopfs/src/test/java/io/lakefs/ImmutablePagination.java

```diff
@@ -0,0 +1,56 @@
+package io.lakefs;
```

Comment:
I don't understand: this used to be auto-generated; is this now manually written?

Reply:
Used the common builder pattern to implement the Pagination class.
This class is used by the test code to pass pagination argumens (some optional).
If Java add optional field initialization I'm sure we didn't had this class in the first place. We do not use the generated code in non-test.